tests/integrations/aws_lambda/test_aws.py

```diff
@@ -49,12 +62,17 @@ def _send_event(self, event):
         # us one.
         print("\\nEVENT: {}\\n".format(json.dumps(event)))
 
-def init_sdk(timeout_warning=False, **extra_init_args):
+    def _send_envelope(self, envelope):
+        envelope = envelope_processor(envelope)
+        print("\\nENVELOPE: {}\\n".format(json.dumps(envelope)))
+
+def init_sdk(timeout_warning=False, traces_sample_rate=0, **extra_init_args):
     sentry_sdk.init(
         dsn="https://123abc@example.com/123",
         transport=TestTransport,
         integrations=[AwsLambdaIntegration(timeout_warning=timeout_warning)],
         shutdown_timeout=10,
+        traces_sample_rate=traces_sample_rate,
```

Comment:
It should not be necessary to pass this explicitly, right? we pass `extra_init_args`

Reply:
Yes, removed this parameter. Change will be in next commit.